application/src/main/java/com/softserve/lv460/application/service/HomeService.java

```diff
@@ -0,0 +1,84 @@
+package com.softserve.lv460.application.service;
+
+import com.softserve.lv460.application.dto.home.HomeRequest;
+import com.softserve.lv460.application.dto.home.HomeResponse;
+import com.softserve.lv460.application.dto.location.LocationResponse;
+import com.softserve.lv460.application.entity.Home;
+import com.softserve.lv460.application.repository.HomeRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@Service
+public class HomeService {
+  @Autowired
```

Comment:
avoid field injection. Use constructor instead. For your convenience, you can use lombok's @AllArgsConstruvtor annotation here and no need to use @Autowired annotation as long as you have only one -args constructor defined in your class